### PR TITLE
This fixes two crashes

### PR DESCRIPTION
**Pull request by analognightmare**
Please review and when you cannot find any crashes anymore then please add the other cards back in and merge :slightly_smiling_face:

### CODE REVIEW

File: src/gui.rs [hunk 1]
```diff
@@ -36,6 +36,13 @@ impl Gui {
         self.cursor_state = c;
     }
 
+    pub fn get_cursor_pos(&self) -> (usize, usize) {
```
Comment (analognightmare) [inline src/gui.rs]: Delete me!

File: src/card.rs [hunk 2]
```diff
@@ -193,11 +197,12 @@ impl CardDeck {
             Coin(1),
             Coin(1),
             Coin(1),
-            Shop,
-            Buff(BuffType::Freeze),
-            Buff(BuffType::Damage),
-            Buff(BuffType::RPM),
-            Buff(BuffType::Range),
+            Coin(1),
```
Comment (analognightmare) [inline src/card.rs]: Delete/uncomment

Comment (eqv) [reply src/card.rs]: done

File: src/gui.rs (hunk 1 above)
Comment (eqv) [reply src/gui.rs]: done

**Pull request merged**